docs/example.ts

```diff
@@ -6,6 +6,7 @@ import Kilt, {
   Message,
   // IRequestForAttestation,
   ISubmitAttestationForClaim,
+  // ICTypeMetadata,
```

Comment:
Honestly, having the Metadata here could be confusing. As it serves no purpose unless storing and retrieving the CType in different languages. 

At the moment, I could remove it. It has its main application within the Demo-client and we have it here to allow people to use it.

Reply:
removed it